ROMFS/px4fmu_common/init.d-posix/1040_standard_vtol

```diff
@@ -34,7 +34,7 @@ then
 
 	param set VT_F_TRANS_DUR 5
 	param set VT_F_TRANS_THR 0.75
-	param set VT_MOT_COUNT 4
+	param set VT_MOT_COUNT 5
```

Comment:
I believe this is the number of MC motors, not total.

Reply:
fair enough